bitnami/postgresql-ha/templates/postgresql/statefulset.yaml

```diff
@@ -4,6 +4,9 @@ metadata:
   name: {{ include "postgresql-ha.postgresql" . }}
   labels: {{- include "postgresql-ha.labels" . | nindent 4 }}
     app.kubernetes.io/component: postgresql
+    {{- with .Values.postgresql.labels }}
+    {{- toYaml . | nindent 4 }}
```

Comment:
Please, could you use `"postgresql-ha.tplValue"`? and have it as a template value too.